platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/maps/NativeMap.java

```diff
@@ -55,6 +55,7 @@ void easeTo(@NonNull LatLng center, double zoom, double bearing, double pitch, d
   @NonNull
   CameraPosition getCameraPosition();
 
+  // Note for implementors: the ordering of the padding is left, top, right, bottom
   CameraPosition getCameraForLatLngBounds(@NonNull LatLngBounds bounds, int[] padding, double bearing, double pitch);
```

Comment:
I have attempted to keep this PR as minimal as possible, but if there's appetite for a deeper API change, I'd suggest that this API should be deprecated and replaced with something unambiguous. In our compose wrapper, we have something with this general shape:

```
data class CameraPadding(
    val left: Int,
    val top: Int,
    val right Int,
    val bottom: Int
)
```